enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -0,0 +1,100 @@
+  vertical-align: top;
+  width: 250px;
+  display: inline-block;
+  margin-left: 20px;
+}
+</style>
+{% endblock %}
+
+{% block extrahead %}
+<script type="text/javascript" src="{% url 'admin:jsi18n' %}"></script>
+<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
+{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+&rsaquo; {% if has_change_permission %}<a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>{% else %}{{ opts.verbose_name_plural|capfirst }}{% endif %}
+&rsaquo; {% if has_change_permission %}<a href="{% url opts|admin_urlname:'change' enterprise_customer.uuid %}">{{ enterprise_customer|truncatewords:"18" }}</a>{% else %}{{ enterprise_customer|capfirst }}{% endif %}
+&rsaquo; {% trans "Manage Learners" %}
+</div>
+{% endblock %}
+
+{% block content %}
+<div class="learners-panel">
+  <h1>Linked learners</h1>
+  <table class="learners-table">
+    <thead>
+      <tr>
+        <th>{% trans "User Email" %}</th>
+        <th>{% trans "Username" %}</th>
+        <th>{% trans "Linked Date" %}</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody>
+    {% for learner in learners %}
+      <tr class="form-row {% cycle 'row1' 'row2' %}" data-email="{{ learner.user_email }}">
+        <td>{{ learner.user_email }}</td>
+        <td>{% if learner.user %}{{ learner.user.username }}{% else %}{% trans "N/A" %}{% endif %}</td>
+        <td>{{ learner.created }}</td>
+        <td class="delete-cell"><input type="button" value="Unlink"></td>
+      </tr>
+    {% endfor %}
+    </tbody>
+  </table>
+</div>
+
+<!-- TODO: find a way to extract it into a file and make edx-platform find that file -->
+<script language="JavaScript" type="text/javascript">
+$(
+  function() {
+    $("table.learners-table .delete-cell input[type='button']").click(function(){
+      var $row = $(this).parents("tr");
+      var email = $row.data('email');
+      var message = interpolate('Are you sure you want to unlink %(email)s?', {'email': email}, true);
+      if (confirm(message)) {
+        $.ajax({
+          url: '?unlink_email='+email,
+          method: "delete",
+          beforeSend: function(xhr) {
+            xhr.setRequestHeader("X-CSRFToken", $.cookie("csrftoken"));
+          }
+        }).success(function() {
+          $row.remove();
+        }).error(function(xhr, errmsg, err){
+          alert(errmsg);
+        })
+      }
+    });
+  }
+);
+</script>
+
+<div class="forms-panel">
+  <h1>{% trans "Link learner" %}</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ manage_learners_form }}
+    <input type="submit" value="Submit" />
+</form>
+</div>
+{% endblock %}
```

Comment:
please add newline. There is a setting in pycharm `Editor > General > Other > 'Ensure line feed at file end on Save'` to automate this.